app/src/test/java/comcmput301f16t01/github/carrier/UserProfileTests.java

```diff
@@ -46,21 +46,21 @@ public void uniqueDriverUsername() {
      */
     @Test
     public void editRiderInfo() {
-        Rider riderOne = new Rider("start username");
-        riderOne.setEmail("start email");
-        riderOne.setPhone("start phone");
+        UserController uc = new UserController();
+        uc.getLoggedInUser().setEmail("start email");
```

Comment:
How did you set the logged in user in this test? This test seems like it will always fail regardless of any implementation because you never set the logged in user?
I suggest using the `uc.setEmail(riderOne, "start email");` instead of using a loggedInUser